holidays/countries/singapore.py

```
@@ -46,7 +46,7 @@ class Singapore(
     References:
         * [Wikipedia](https://en.wikipedia.org/wiki/Public_holidays_in_Singapore)
         * [Holidays Act 1998](https://web.archive.org/web/20250405061431/https://sso.agc.gov.sg/Act/HA1998)
-        * [Ministry of Manpower](https://web.archive.org/web/20250424152111/https://mom.gov.sg/employment-practices/public-holidays)
+        * [Ministry of Manpower](https://web.archive.org/web/20250616105633/https://mom.gov.sg/employment-practices/public-holidays)
```

Pylint (3.3.7)

[convention] Line too long (132/100)

(C0301)